ql/src/java/org/apache/hadoop/hive/ql/io/arrow/ArrowColumnarBatchSerDe.java

```diff
@@ -185,7 +186,7 @@ private static Field toField(String name, TypeInfo typeInfo) {
         final TypeInfo keyTypeInfo = mapTypeInfo.getMapKeyTypeInfo();
         final TypeInfo valueTypeInfo = mapTypeInfo.getMapValueTypeInfo();
         final StructTypeInfo mapStructTypeInfo = new StructTypeInfo();
-        mapStructTypeInfo.setAllStructFieldNames(Lists.newArrayList("keys", "values"));
+        mapStructTypeInfo.setAllStructFieldNames(Lists.newArrayList("key", "value"));
```

Comment:
As the representation of map datatype in ArrowColumnarBatchSerDe is like a list of structs data-type, where each struct element is the key-value pair of the map. 
So the structFieldNames precisely should be "key" and "value", instead of keys and values.

Reply:
Make sense.